core/core.go

```diff
@@ -408,7 +445,7 @@ func (c *Core) WriteBlock(block *types.Block) {
 		}
 		nodeCtx := common.NodeLocation.Context()
 		if order == nodeCtx {
-			parentHeader := c.GetHeader(block.ParentHash(), block.NumberU64()-1)
+			parentHeader := c.GetBlock(block.ParentHash(), block.NumberU64()-1)
```

Comment:
Why did you change this?

Reply:
good question, will change, unnecessary to read the block, good catch